src/pudl/metadata/fields.py

```diff
@@ -1415,6 +1411,14 @@
         "description": "Net plant capability in megawatts.",
         "unit": "MW",
     },
+    "plant_classification_type": {
+        "type": "string",
+        "description": "Status of electric plant (in service, future, leased, total).",
+    },
+    "plant_functional_type": {
+        "type": "string",
+        "description": "Functional plant type (steam, hydro, distribution, etc.).",
+    },
```

Comment:
can we rename these to `plant_status` and `plant_type` or `asset_type` or seomthing